### DISCUSSION TITLE
Two PowerGrid DataTable in One Page, Doing sorting event another table will disappear

### DISCUSSION THREAD

**Discussion by shiroamada** [Power-Components/livewire-powergrid#391] [Q&A]
How to make two powergrid datatable work together in one page?

It is working well when initial data display, but I did filter or sorting. The 2nd Datatable will be disappear.

Anyway I can two tables working well together without disturb the second table.

**Comment by luanfreitasdev** [comment]
Hello. Could you show the behavior on video?

**Comment by shiroamada** [comment]
Sure

```
<h3 class="text-green-500">NEW ORDERS</h3>
<livewire:ready-to-pick-up />

<div class="my-8"></div>

<h3 class="text-red-500">READY TO PICK UP</h3>
<livewire:pick-up-table />

```

https://user-images.githubusercontent.com/2062946/162996535-9042e4ec-dc3a-4ccc-a234-7472055b7564.mp4

**Reply by luanfreitasdev** [reply]
https://livewire-powergrid.com/#/get-started/use-your-powergrid-table?id=passing-arguments

```html
<h3 class="text-green-500">NEW ORDERS</h3>
<livewire:ready-to-pick-up tableName="table1" /> // for example

<div class="my-8"></div>

<h3 class="text-red-500">READY TO PICK UP</h3>
<livewire:pick-up-table tableName="table2" /> // for example
```

**Comment by shiroamada** [comment]
I added, but still not working.


https://user-images.githubusercontent.com/2062946/163006654-148ad008-fa43-4d33-816b-88e48f31ac6e.mp4

**Comment by shiroamada** [comment]
composer.json information
<img width="583" alt="image" src="https://user-images.githubusercontent.com/2062946/163008241-485c1c43-402d-4385-b37c-0fe024d8c5c5.png">

**Reply by luanfreitasdev** [reply]
There is no error in the console?. Have you tried using wire:key?

**Reply by shiroamada** [reply]
No error in console.
I notice if i do search in bottom table, then top table will not affected. But if I search at the top table, the bottom table will disappear.

if I add wire:key then it is work at the top table but console have error, but the bottom table will throw Live WireException
```
                    <div wire:key="foo">
                    <livewire:ready-to-pick-up tableName="ready-to-pick-up" :bex_db="$bex_db" :code="$code" :pick_up_from_location="$pick_up_from_location" />
                    </div>

                    <div class="my-8"></div>


                    <h3 class="text-red-500">READY TO PICK UP</h3>
                    <div wire:key="bar">
                    <livewire:pick-up-table tableName="pick-up" :bex_db="$bex_db" :code="$code" :pick_up_from_location="$pick_up_from_location" />
                    </div>
```

Console Error:
<img width="323" alt="image" src="https://user-images.githubusercontent.com/2062946/163075496-2dc5b15a-de0f-4052-9ed5-906ec7337a7d.png">
